pkgs/tools/security/enpass/update_script.py

```diff
@@ -0,0 +1,95 @@
+from __future__ import print_function
```

Comment:
Looks a little bit complicated for a simple version update script, but if it floats the boat...